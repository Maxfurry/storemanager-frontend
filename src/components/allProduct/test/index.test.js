import React from 'react';
import { shallow } from 'enzyme';
import Products from '..';

describe('Products', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Products />);
    expect(wrapper).toMatchSnapshot();
  });
});
