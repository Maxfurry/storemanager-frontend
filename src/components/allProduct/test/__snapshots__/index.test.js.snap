// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Products should match snapshot 1`] = `
<div
  className="productContainer"
>
  <div>
    <p>
      Home
    </p>
    <h1>
      Popular
    </h1>
  </div>
  <div
    className="myProducts"
  >
    <Card
      className="myCard"
    >
      <Image
        as="img"
        className="previewImage"
        ui={true}
      />
      <CardContent
        className="productDetails"
      >
        <CardHeader
          className="productTitle"
        />
        <CardMeta
          className="productPrice"
        >
          <span
            className="date"
          />
        </CardMeta>
        <Button
          animated="vertical"
          as="button"
          className="addToCart"
        >
          <ButtonContent
            visible={true}
          >
            Shop
          </ButtonContent>
          <ButtonContent
            hidden={true}
          >
            <Icon
              as="i"
              name="shop"
            />
          </ButtonContent>
        </Button>
      </CardContent>
    </Card>
  </div>
</div>
`;

exports[`Products should match snapshot 2`] = `
<div
  className="productContainer"
>
  <div>
    <p>
      Home
    </p>
    <h1>
      Popular
    </h1>
  </div>
  <div
    className="myProducts"
  />
</div>
`;
